gst/nnstreamer/tensor_source/tensor_src_iio.c

```diff
@@ -147,6 +147,8 @@ enum
 #define EN_SUFFIX "_en"
 #define INDEX_SUFFIX "_index"
 #define TYPE_SUFFIX "_type"
+#define SCALE_SUFFIX "_scale"
+#define OFFSET_SUFFIX "_offset"
```

Comment:
This macro actually is not used in this code. 
If it is going to used in the near future, then that's OK.

By the way, are those suffixes the standard format in IIO world?

Reply:
Corrected.

I am creating the suffixes to access the files containing the corresponding details (index, type, scale etc) for all the channels.
These suffixes are shared by all the channels as these attributes are either present for all the channels individually, or there is just one set of these params in a generic file which is shared by all the channels.

Yes, these suffixes and the generated file names are as per standard. (https://www.kernel.org/doc/Documentation/ABI/testing/sysfs-bus-iio contains more info)